### DISCUSSION TITLE
mail push

### DISCUSSION THREAD

**Discussion by orlovnv** [Z-Hub/Z-Push#124] [Q&A] [answered]
is it possible to speed up push receiving, for example email came to inbox and after 1 minute I receive push

**Comment by orlovnv** [comment]
z-push stops sync folders 30 sec   after recieve email and if new email come it doesn't push to phone

**Reply by orlovnv** [reply]
please help, push begin work if I open mail or calendar on my phone, then I recieve push

**Reply by matidau** [reply, marked as answer]
Hi again,

Can you check your settings on your phone and ensure that it is set to automatic sync for this account. This is called different things on Android and iPhone.

After that can you run z-push-top and see if the phone is pinging the server, especially when you find it is delayed or stops syncing.
